### DISCUSSION TITLE
Why i can't use the inpaint or outpaint

### DISCUSSION THREAD

**Discussion by chenhaokui** [lllyasviel/Fooocus#3051] [Q&A]
![Snipaste_2024-05-31_00-06-03](https://github.com/lllyasviel/Fooocus/assets/92264358/5d7648c6-1cd2-4d18-9abe-16e19c99b8d5)

**Comment by mashb1t** [comment]
@chenhaokui what's the exact issue? please elaborate.

**Comment by chenhaokui** [comment]
It loads,and fail in the end
---- Replied Message ----
FromManuel ***@***.***>Date05/31/2024 17:22 ***@***.***> ***@***.***>,
***@***.***>SubjectRe: [lllyasviel/Fooocus] Why i can't use the inpaint or outpaint (Discussion #3051)
@chenhaokui what's the exact issue? please elaborate.

**Reply by mashb1t** [reply]
@chenhaokui please provide the full terminal log output for better debugging. I assume your haedware is too low speced, but can't tell wothout full logs.
